### ISSUE TITLE
Addition Of Exact Resources Link

### ISSUE THREAD

**Issue by Mannuel25** [jobream/List-of-Learning-Resources#94]
I'd suggest the appropriate links that points out how to learn a particular think are used. For instance, if I wanna learn Python any website link I click the the Markdown files should direct me to exactly where to learn Python in the website.

More explanation:
(Tutorialspoint)[https://www.tutorialspoint.com/python/index.htm], if I add Tutorialspoint resource, when I click the link, it should lead me to where to learn Python on Tutorialspoint's website.

**Comment by MRslimcoder** [comment]
@Mannuel25 for that we need to verify each and every link <3

**Comment by Mannuel25** [comment]
> @Mannuel25 for that we need to verify each and every link <3

Yes, any link added to learn a particular thing should take the user exactly to where it can be learnt in the platform or website.

**Comment by Mannuel25** [comment]
@MRslimcoder , I opened a new issue on the updated of the README. If I'm permitted, I'll take care of the links :)

**Comment by MRslimcoder** [comment]
#98 are you talking about this issue if yes then go ahead and send us PR.

**Comment by Mannuel25** [comment]
> #98 are you talking about this issue if yes then go ahead and send us PR.

Okay, I'll work on it and make a PR soon.
